alternative-log-storage/README.md

```diff
@@ -0,0 +1,11 @@
+## Alternative Log Storage
```

Comment:
Maybe the folder name only `log-storage`

Add short info in the main readme with reference to here

Reply:
Add an example how to use it as module

Reply:
4a3b78067256fa82e639444b5e6cee646c0df1f3